CardScan/Classes/Models.swift

```diff
@@ -99,7 +84,7 @@ class FindFour {
     
     /// Construct a model that automatically loads the model from the app's bundle
     convenience init() {
-        try! self.init(contentsOf: type(of:self).urlOfModelInThisBundle)
+        try! self.init(contentsOf: BundleURL)
```

Comment:
Do we need this still? We can just instantiate the class directly using the url we get from your static method